src/js/Controllers/RpcFactory.js

```diff
@@ -148,6 +148,65 @@ class RpcFactory {
             }
         })
     }
+    static UIShowResponse(rpc) {
+        var supportedTemplates = capabilities["MEDIA"].displayCapabilities.templatesAvailable;
+        const templateConfiguration = rpc.params.templateConfiguration;
+        const templateExists = templateConfiguration 
+            && templateConfiguration.template ? true : false;
```

Comment:
```suggestion
        const templateExists = templateConfiguration && templateConfiguration.template;
```
seems this is a boolean already